<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicListClass</class>
 <widget class="QWidget" name="PicListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像列表</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Open">
       <property name="text">
        <string>打开图片</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Del">
       <property name="text">
        <string>删除图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save">
       <property name="text">
        <string>保存图片</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Fog">
       <property name="text">
        <string>加雾</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Rain">
       <property name="text">
        <string>加雨</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Snow">
       <property name="text">
        <string>加雪</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QListWidget" name="lw_Pics"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
